docs/docs/upgrade/network-api-migration.md

```diff
@@ -213,6 +216,9 @@ The following table summarizes the API transformations when using the network AP
 
  {
     “container”: {
+      "docker": {
+        "image": "image-name"
```

Comment:
please include the `image` field in the `docker` object in the first column as well.